clients/src/main/java/org/apache/kafka/clients/consumer/internals/DefaultBackgroundThread.java

```diff
@@ -178,46 +169,34 @@ public void run() {
      * 3. Poll the networkClient to send and retrieve the response.
      */
     void runOnce() {
-        drain();
```

Comment:
Reviewer's note: consolidated a lot of small methods into the `runOnce` method for clarity.